.github/workflows/release-workflow-dispatcher.yml

```diff
@@ -0,0 +1,266 @@
+name: Release Workflow Dispatcher
+on:
+  workflow_dispatch:
+    inputs:
+      openebs-crds-version:
+        required: true
+        default: ""
+      localpv-provisioner-version:
+        required: true
+        default: 4.2.0
```

Comment:
Will you change the default after every release? Or it's just for example? Shoudn't we just error out if this is empty rather than defaults?

Reply:
No, its just an example of how to set values. All parameters are required. If not provided/empty the workflow won't run and will ask the user to populate the field.

Reply:
So it won't show up as the default value when running the workflow and user still has to manually type it? If so then it's good, otherwise I agree with Abhinandan, I think better to leave empty to avoid mistakes

Reply:
I'll remove the default values and make them empty.